plugins/lighthouse-backend/package.json

```diff
@@ -24,7 +24,8 @@
     "lighthouse"
   ],
   "files": [
-    "dist"
+    "dist",
+    "config.d.ts"
   ],
```

Comment:
Hey you forgot to also add a `"configSchema": "config.d.ts"`, [like this](https://github.com/backstage/backstage/blob/9875cfb30b7ca5ae2047472201337b30796068fb/plugins/kubernetes-backend/package.json#L39). Otherwise the config won't get discovered.

Reply:
Oh sorry, fixing that

Reply:
Thank you very much for the review.